tasks/package.json

```diff
@@ -0,0 +1,5 @@
+{
+    "dependencies": {
+        "ethers": "6.7.1"
```

Comment:
huh??

Reply:
Using it for getting superTokenLogic contract addresses from superTokenFactory contracts as suggested by @0xdavinchee

Reply:
okay, I don't like that we do it here. Let me suggest another way.